src/Factories/UpdateProductControllerFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace App\Factories;
+
+
+use App\Controllers\updateProductController;
```

Comment:
wrong namespace? I think this is my bad from my update story though, sorry!
```suggestion
use App\Controllers\UpdateProductController;
```